Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/STSWebIdentityAWSCredentialIdentityResolver.swift

```diff
@@ -240,6 +261,20 @@ public actor STSWebIdentityAWSCredentialIdentityResolver: AWSCredentialIdentityR
         return try String(contentsOfFile: resolvedPath, encoding: .utf8)
             .trimmingCharacters(in: .whitespacesAndNewlines)
     }
+
+    private func resolveTokenFeatureID() -> [String] {
+        switch source {
+        case .env:
+            return [CredentialFeatureID.CREDENTIALS_ENV_VARS_STS_WEB_ID_TOKEN.rawValue]
+        case .configFile:
+            return [CredentialFeatureID.CREDENTIALS_PROFILE_STS_WEB_ID_TOKEN.rawValue]
+        case .mixed:
+            return [
+                CredentialFeatureID.CREDENTIALS_ENV_VARS_STS_WEB_ID_TOKEN.rawValue,
+                CredentialFeatureID.CREDENTIALS_PROFILE_STS_WEB_ID_TOKEN.rawValue
+            ]
+        }
+    }
```

Comment:
Helper function to determine feature IDs related to where STS web identity token was resolved from.